alphadia/calibration/models.py

```diff
@@ -118,26 +124,23 @@ def _kernel_indices_density(self, x: np.ndarray):
 
         return np.column_stack([start, end]).astype(int)
 
-    def fit(self, x: np.ndarray, y: np.ndarray):
-        """fit the model passed on provided training data.
+    def fit(self, x: np.ndarray, y: np.ndarray) -> "LOESSRegression":  # noqa: C901, PLR0912 Too complex, too many branches
```

Comment:
fun fact, this is one of the oldest pieces of code where I tried to write "software"